tests/phpunit/tests/post/filtering.php

```diff
@@ -17,11 +17,6 @@ function setUp() {
 
 	}
 
-	function tearDown() {
-		kses_remove_filters();
```

Comment:
I agree that this function call is useless as `WP_UnitTestCase_Base::_restore_hooks()` in `WP_UnitTestCase_Base::tearDown()` is called afterwards and may, in effect, "undo" what this function does.
https://developer.wordpress.org/reference/functions/kses_remove_filters/.

Having said that, it being in `tearDown()` was probably the wrong choice in the first place. I have a feeling that this call should be moved to the `setUp()` method for this class instead.

Reply:
What this function is doing is the opposite of `kses_init_filters()` which is called in the `setUp()` method of this class to initialise KSES for the tests. As you mentioned, its functionality is handled by the `_restore_hooks()` method.